.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "libs/QGumboParser"]
```

Comment:
would it be possible to use a regex for parsing the pages instead? would like to avoid adding more dependencies if possible

Reply:
unfortunately, code becomes much more complicated
probably we can use the gumbo parser as embedded code instead of submodule
I have strong feelings it will be useful in future